dashboard/config/marketing/announcements.json

```diff
@@ -37,6 +37,15 @@
       "buttonText": "Learn more",
       "buttonUrl": "https://code.org/lms",
       "buttonId": "lms-launch"
+    },
+    "curriculum-launch-2024": {
```

Comment:
Adding this preemptively so the strings can get in the pipeline. Jira ticket: [ACQ-1968](https://codedotorg.atlassian.net/browse/ACQ-1968)